# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../bin86/fix_ftbfs.patch
# Copyright (C) 2012 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

Fix "FTBFS: ncc: illegal label": apply patch by dai that replaces strcpy by memmove.
http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=591133

--- linux86-0.16.17/copt/copt.c.orig	2003-10-08 04:46:35.000000000 +0900
+++ linux86-0.16.17/copt/copt.c	2010-10-12 12:30:25.000000000 +0900
@@ -174,7 +174,7 @@ static char *readline(FILE *fp)
   /* Delete leading white spaces */
   for (cp = buf; *cp && isspace(*cp); cp++) ;
   if (cp != buf && *cp)
-	strcpy(buf, cp);
+	memmove(buf, cp, strlen(cp) + 1);
 
   return(buf);
 }

